pkg/kdk/destroy.go

```diff
@@ -15,26 +15,25 @@
 package kdk
 
 import (
-	"context"
 	"fmt"
+
 	"github.com/Sirupsen/logrus"
-	"github.com/cisco-sso/kdk/internal/pkg/utils/simpleprompt"
+	"github.com/cisco-sso/kdk/pkg/prompt"
 	"github.com/docker/docker/api/types"
-	"github.com/docker/docker/client"
 )
 
-func Destroy(ctx context.Context, dockerClient *client.Client, logger logrus.Entry) error {
+func Destroy(cfg KdkEnvConfig, logger logrus.Entry) error {
```

Comment:
Should use `c` instead of `cfg` to stay consistent with other go files in this project.